constitution.md

```diff
@@ -0,0 +1,232 @@
+rights for high-level directives as set by the administrators of a given project.
+
+Anyone can become a developer by signing up and *signing the CLA*. In exchange for voting rights, a developer must commit to 
+5 hours per month of development work on any Chisel project. This work can include:
+* creating pull requests
+* reviewing pull requests
+* submitting issues
+* discussing issues
+* answering Stack Overflow questions
+* attending meetings
+* external contributions (registered on website via submissions)
+  * writing blogs
+  * additional documentation
+* red lighting/green lighting releases
+
+Developers who have been inactive for six months or more will be removed from the group.
+
+A developer interested in becoming a reviewer should have a wide body of commits and begin reviewing submitted pull 
+requests to practice and build up a body of reviews, both to demonstrate their expertise but also to increase their 
+familiarity with the repository. These pull-requests will also require reviews from a Reviewer. Once a developer has 
+had over 10 reviews on a project, they can submit a request to the administrators to become a Reviewer for that project.
+
+Developers can contribute feedback by testing scheduled releases and fixes, and these members are critical to ensuring 
+Chisel's backwards compatibility, as well as discovering bugs before a release. 
+
+### Volunteers
+Volunteers are people who want to contribute to a Chisel project, but do not commit to a regular contribution. They do 
+not have voting rights, and no registration is required. Volunteers may attend development meetings. Volunteers may 
+contribute through PR’s to Chisel repositories, and are encouraged to review and comment but they cannot accept PR requests. It is hoped that volunteers will become developers.
+
+## Collaboration Policy
+CWG’s standard communication channels and development policy is outlined as follows, and will likely be adopted for other 
+associated CHIPS Alliance working groups.
+
+## Communication Channels
+High level leadership directives will be published via mailing list, and minutes of these discussions will be posted on the 
+Chisel website.
+
+All technical development discussions will be done either on Github with Github issues, pull requests, and comments, or via 
+a recurring open development meeting with video conferencing. Open development meetings will be run by project 
+administrators or senior reviewers.
+
+User-facing support is primarily through answering questions on **Stack Overflow**, contributing to and maintaining a **Github 
+Wiki page**, and promoting community dialogue with an associated **Gitter** channels. The purpose of Gitter is to move discussion 
+from mailing lists to a more modern communication outlet.
+
+
+### Development Policy
+Generally, development on mature products will release minor/major versions, where minor versions are backwards compatible, 
```

Comment:
Should we explicitly reference semantic versioning?

Reply:
Ok to be high level in the constitution, as changes to versioning likely shouldn't require a change to the constitution.